source/containers.tex

```diff
@@ -2522,7 +2522,7 @@
 &   \tcode{void}
 &   \requires\ \tcode{value_type} shall be \tcode{EmplaceConstructible} into \tcode{X} from \tcode{*i}.\br
     \requires \tcode{i} and \tcode{j} are not iterators in \tcode{a}.
-    Equivalent to \tcode{a.insert(t)} for each element in \tcode{[i,j)}.%
+    \effects Equivalent to \tcode{a.insert(t)} for each element in \tcode{[i,j)}.%
```

Comment:
Wait, did you test this? You'll probably need a manual  line break of sorts here.

Reply:
No, I didn't. Guess I'll setup a proper latex editing environment with live rendering.